### ISSUE TITLE
Unable to log in Tools --> Settings page

### ISSUE THREAD

**Issue by rootchick** [pddpauw/BirdPi#6]
Hi there!  I've installed on my pi 5 with your script (thank you!!) and I can access the web interface, and can log into the Tools page with username=birdnet and no password, but I am unable to log into the Settings page.  I've tried setting a plaintext password with only alphanumeric characters in /etc/birdnet/birdnet.conf (there was a bug report in the original repo about not being able to use non-alphanumeric passwords) under CADDY_PWD=.  I've also tried hashing a password with 'caddy-password' and pasting it in there.  I'm no caddy expert, I just saw that suggestion in another thread somewhere.  I'm also not finding that it logs any errors anywhere.  There is no syslog and it appears to be configured to log to /var/log/php8.2-fpm.log but I haven't seen any pertinent errors there, even though it looks like logging is enabled in the php.ini file.

If I cancel the login box on the Settings page, I get a "You cannot edit the settings for this installation" error.  

Any help would be appreciated, thanks!

**Comment by WillKrzy** [comment]
Same Issue. Please Fix!

**Comment by anthonylhill** [comment]
Still a problem.  I'll happily test any suggestions.

**Comment by farmhandy** [comment]
yes, same here, is there a solution or just re install, i really dont want to do that ive so many other things setup on the rpi.

**Comment by pddpauw** [comment]
I've stopped maintaining this one, please use this: https://github.com/Nachtzuster/BirdNET-Pi
